### PR TITLE
Adjustment for MyInvision style

### PR DESCRIPTION
**Pull request by MannixMD**

### CODE REVIEW

File: styles/MyInvision/theme/mchat_custom.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    top: -10px;
+}
+.mchat-wrapper .header {
+    padding-left: 15px;
+    padding-top: 10px;
+}
+#mchat-panel {
+    border-bottom: 1px solid #ccc;
+    border-left: 1px solid #ccc;
+    border-right: 1px solid #ccc;
```
Comment (kasimi) [inline styles/MyInvision/theme/mchat_custom.css]: Please use tabs for indentation.

**Review by kasimi — CHANGES_REQUESTED** [review]
Thanks for the PR. Can you please convert your spaces to a tab?

File: styles/MyInvision/theme/mchat_custom.css (hunk 1 above)
Comment (kasimi) [reply styles/MyInvision/theme/mchat_custom.css]: Also, in the style's `style.cfg` it says `name = myInvision`. Will it work with `MyInvision` (upper-case M) as folder name?

Comment (MannixMD) [reply styles/MyInvision/theme/mchat_custom.css]: I've tested it on my test board and worked fine. Fixed the spaces

Comment (kasimi) [reply styles/MyInvision/theme/mchat_custom.css]: There are now 8 spaces and no tab. Maybe it's your editor that inserts spaces when you hit tab?

**Review by kasimi — APPROVED** [review]

**Pull request merged**